examples/selection.rs

```diff
@@ -143,6 +146,8 @@ fn main() {
             }
         }
     });
+    // Get primary selection manager
+    env.get_primary_selection_manager();
```

Comment:
This is not something ideal, so I wonder how we should handle such init of primary selection, since it's done as LazyGlobal, like `Shell`, for `Shell` it's not a problem, since you're asking for configure/creating surface, but for selection it could be desired to initialize it transparently in a calls on a helper.

Reply:
I'd suggest we add this call into the `init_default_environment!()` macro, and add a note in the module-level docs about the need to do that for users not using the macro.